test/ast.js

```diff
@@ -286,6 +286,59 @@ describe('AST', () => {
       'Receive Ether functions cannot have return parameters');
   })
 
+  it('FunctionDefinition with one override', () => {
+    var ast = parseNode("function foo() public override(Base) {}")
```

Comment:
Should you add another FunctionDefinition with one override, but no arguments?

 i.e. `function foo() public override {}`

Reply:
Yes! I will add it, thank you.